compiler/luci/partition/src/CircleNodeClone.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO consider relocate this file
```

Comment:
I think `luci/service/include/luci/Service/CircleNodeClone.h` seems good because

- `luci/service/include/luci/Service/Nodes/CircleConst.h` already have same function
- cloning is kind of basic **service** of luci

However, integration may require additional workloads, we may go as it is and later we may relocate/refactor both this file and `luci/service/include/luci/Service/Nodes/CircleConst.h` together.
It is okay?

Reply:
> It is okay?

The comment is added as you expected... I'm not sure yet where to relocate...
Um... but what do you mean by "it is okay?" ?

Reply:
"Following two things are okay?" was my question :)

- Go as it is
- And relocate this file to `luci/service/include/luci/Service/CircleNodeClone.h`